packages/hash/api/src/graphql/resolvers/pages/setParentPage.ts

```diff
@@ -1,87 +1,121 @@
 import { ApolloError } from "apollo-server-express";
 import { MutationSetParentPageArgs, Resolver } from "../../apiTypes.gen";
 import { LoggedInGraphQLContext } from "../../context";
-import { Entity, Page, UnresolvedGQLEntity } from "../../../model";
+import { DBClient } from "../../../db";
+import { Entity, Page, UnresolvedGQLEntity, User } from "../../../model";
 import { topologicalSort } from "../../../util";
 
+async function deleteParentPage(
```

Comment:
These should both be methods on the `Page` model class, so their functionality can be reused in other parts of the codebase. I'm happy for you not to do this as part of this PR, I'm planning on reconciling this as part of [this PR](https://github.com/hashintel/hash/pull/364/files).

Reply:
I would like to get this PR in so that Teenoh will have an easier time doing the sidebar implementation. I would appreciate you reconciling this in the linked PR :)

Reply:
Understood, I'm happy for this to be left as is - just want to point out I will be moving this logic into the `Page` model class very soon.

Reply:
Resolved by moving logic into `Page` model class